README.md

```diff
@@ -77,6 +79,103 @@ spec:
         value: some-value
 ```
 
+### EKS Fargate sample spec
+The purpose of this provisioner is to enable instance groups with AWS Fargate management. 
+
+By associating EKS clusters with a Fargate Profile, pods can be identified for execution through profile selectors. If a to-be-scheduled pod matches any of the selectors in the Fargate Profile, then that pod is scheduled on Fargate. 
+
+An EKS cluster can have multiple Fargate Profiles. If a pod matches multiple Fargate Profiles, Amazon EKS picks one of the matches at random.  
+
+EKS supports clusters with both local worker nodes and Fargate management.  If a pod is scheduled and matches a Fargate selector then Fargate manages the pod.  Otherwise the pod is scheduled on a worker node.  Clusters can be defined without any worker nodes (0) and completely rely upon Fargate for scheduling and running pods. 
+
+More on [Fargate](https://docs.aws.amazon.com/eks/latest/userguide/fargate.html).
+
+The provisioner will manage (create and delete) Fargate Profiles on any EKS cluster (within the account) regardless of whether the cluster was created via CloudFormation, the AWS CLI or the AWS API. 
+
+Below is an example specification for the **eks-fargate** provisioner
+
+```yaml
+apiVersion: instancemgr.keikoproj.io/v1alpha1
+kind: InstanceGroup
+metadata:
+  name: hello-world
+spec:
+  # provision for EKS using Fargate
```

Comment:
Will this spec work? I think CRD mandates having spec.strategy

Reply:
whoops.  you are right.  adding...